terra_notebook_utils/table.py

```diff
@@ -215,13 +216,21 @@ def list_rows(table: str, **kwargs) -> Generator[Row, None, None]:
             yield Row(item['name'], _attributes_from_fiss_response(item['attributes']))
         page_number += 1
 
-def get_row(table: str,
-            entity_id: str,
-            workspace_name: Optional[str]=WORKSPACE_NAME,
-            workspace_google_project: Optional[str]=WORKSPACE_GOOGLE_PROJECT):
-    resp = fiss.fapi.get_entity(workspace_google_project, workspace_name, table, entity_id)
-    resp.raise_for_status()
-    return resp.json()
+def get_row(table: str, item: ROW_OR_NAME, **kwargs) -> Optional[Row]:
```

Comment:
Echoing an earlier comment:

```
Nit: Grab bag `**kwargs` are kind of unfriendly.  It avoids typing and documentation about what inputs are expected.  Consider putting this information into the function docstring if it's unavoidable.
```

I'm not sure if your style is changing but this is similar to Hannes's style if you take a look at Azul.  This is also something that was throughout Toil when I first joined.  It's not so unclear when used sparingly, but... I have particularly strong memories of how confusing it can be when over-used, especially if lacking docstrings (or worse, the docstrings are outdated and actually document the wrong kwargs).